sql/src/main/java/io/crate/metadata/table/ColumnRegistrar.java

```diff
@@ -54,17 +57,44 @@ public ColumnRegistrar(RelationName relationName, RowGranularity rowGranularity)
         this.rowGranularity = rowGranularity;
         this.infosBuilder = ImmutableSortedMap.naturalOrder();
         this.columnsBuilder = ImmutableSortedSet.orderedBy(Reference.COMPARE_BY_COLUMN_IDENT);
+        this.expressionBuilder = ImmutableSortedMap.naturalOrder();
     }
 
-    public ColumnRegistrar register(String column, DataType type) {
-        return register(new ColumnIdent(column), type);
+    public ColumnRegistrar<T> register(String column, DataType type) {
```

Comment:
should DataType be generic here and in two methods bellow?

Reply:
no, this is on purpose to be temporally backward compatible until all static tables are moved to the new approach.